src/Process/3dTilesProcessing.js

```diff
@@ -96,7 +96,6 @@ export function pre3dTilesUpdate(context, layer) {
 function computeNodeSSE(camera, node) {
     if (node.boundingVolume.region) {
         const worldCoordinateCenter = node.boundingVolume.region.centerWorld;
-        worldCoordinateCenter.applyMatrix4(node.matrixWorld);
```

Comment:
This won't work if your 3d-tiles object is attached to an object with a non-identity transform.
That's because `centerWorld` isn't in world coords (aka three.js coords): it's in WGS84-world-coords.